File: Dataset/Leetcode/test/58/580.cpp
class Solution {
public:
    int XXX(string s) {
        int cnt = 0;
        for(int i = s.length() -1; i >= 0;i--){
            if(s[i] != ' ') {
                cnt++;
            } else if(cnt > 0) {
                return cnt;
            }
        }
        return cnt;
    }
};

File: Dataset/Leetcode/train/78/580.py
 class Solution(object):
    def XXX(self, nums):
        """
        :type nums: List[int]
        :rtype: List[List[int]]
        """
        res = [[]]
        for num in nums:
            print(num)
            res += [x+[num] if x else [num] for x in res]
        return res

